source/class/qx/tool/cli/api/Test.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * This is used by listeners of the "runTests" event
+ */
+qx.Class.define("qx.tool.cli.api.Test", {
+  extend: qx.core.Object,
+  construct: function(name) {
+    this.base(arguments);
+    this.setName(name);
+  },
+  properties: {
+    name: {
+      check: "String",
+      event: "changeName"
+    },
+    description: {
+      check: "String",
+      event: "changeDescription"
+    },
+    exitCode: {
+      check: "Number",
+      event: "changeExitCode",
+      nullable: true,
```

Comment:
Should be 0 as default and not nullable. It's the process exit code should be a  valid value.

Reply:
If it is 0 then you cannot listen for the changeExitCode event in case of a successful test. It needs to be in some "undefined" state to signal the test hasn't run yet. If it is not set, the test hasn't run.